src/System Application/App/Company Initialize/CompanyInitialize.Table.al

```diff
@@ -0,0 +1,33 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Environment.Configuration;
+
+table 900 "Company Initialize"
+{
+    DataClassification = SystemMetadata;
+
+    fields
+    {
+        field(1; "Primary Key"; Integer)
+        {
+        }
+        field(2; "Initialized Time"; DateTime)
+        {
+        }
+        field(3; "Initialized Version"; Code[20])
+        {
+        }
```

Comment:
One thing I'd add here is a field: `IsCompanyInitialized` (or something like that)

If you look at some code in BaseApp, we are leveraging the existences of GLSetup to determine this

Reply:
Yes, I was using the fact that the table is empty as it not being initialized. Do you have a suggestion for when we want to insert a record but state that the company has not been initialized?